web/turkey/survey/static/survey/js/auditors/clicks_specific.js

```diff
@@ -0,0 +1,33 @@
+var AuditorClicksSpecifics = {
+    clicks_specifics: [],
+    log_click_content: function (e) {
+        var dom = {
+            'type'  :   e.target.nodeName.toLowerCase(),
+            'id'    :   e.target.id != '' && e.target.id != undefined 
+                            ? e.target.id : null,
+            'class' :   e.target.class != '' && e.target.class != undefined 
+                            ? e.target.class : null,
+            'name'  :   e.target.name != '' && e.target.name != undefined 
+                            ? e.target.name : null                
+        };
+        this.clicks_specifics.push(dom);
+    },
+    submit_callable: function () {
+        return {
```

Comment:
This method of return can be altered in a way that prevents you from having to override base class behavior when you get to the Python stage. Change is simple. Instead of returning an Object, which will be converted to a dictionary, return an array of Objects with key/values corresponding to your eventual data model.

SO all you really need to do is 
`return this.clicks_specifics`

See _EventAndSubmissionModel.save_processed_data_to_model for more detail if desired